index.js

```diff
@@ -46,7 +45,7 @@ module.exports = class ServeDrive extends ReadyResource {
     for (const c of this.connections) {
       c.destroy()
     }
-    if (this.server.listening) {
+    if (this.opened) {
```

Comment:
If `_close` is called, isn't `opened` always set?

Reply:
only if ready is called first, otherwise you couldn't need this line at all i guess.